django/mgmt/views.py

```diff
@@ -590,6 +602,11 @@ def get(self, request, collection_name, experiment_name, exp_form=None, chan_for
         if err:
             return err
 
+        is_admin_int = 0
+        is_admin = BossPermissionManager.is_in_group(request.user, ADMIN_GRP)
+        if is_admin:
+            is_admin_int = 1
```

Comment:
Was there an issue passing `is_admin` to the template? We want to use a boolean value rather than an integer because that's the idiomatic way of conveying this information.